price_rule.go

```diff
@@ -51,25 +51,25 @@ type PriceRule struct {
 	PrerequisiteCollectionIds              []int64                                 `json:"prerequisite_collection_ids,omitempty"`
 	PrerequisiteSavedSearchIds             []int64                                 `json:"prerequisite_saved_search_ids,omitempty"`
 	PrerequisiteCustomerIds                []int64                                 `json:"prerequisite_customer_ids,omitempty"`
-	PrerequisiteSubtotalRange              *prerequisiteSubtotalRange              `json:"prerequisite_subtotal_range,omitempty"`
-	PrerequisiteQuantityRange              *prerequisiteQuantityRange              `json:"prerequisite_quantity_range,omitempty"`
-	PrerequisiteShippingPriceRange         *prerequisiteShippingPriceRange         `json:"prerequisite_shipping_price_range,omitempty"`
-	PrerequisiteToEntitlementQuantityRatio *prerequisiteToEntitlementQuantityRatio `json:"prerequisite_to_entitlement_quantity_ratio,omitempty"`
+	PrerequisiteSubtotalRange              *PrerequisiteSubtotalRange              `json:"prerequisite_subtotal_range,omitempty"`
+	PrerequisiteQuantityRange              *PrerequisiteQuantityRange              `json:"prerequisite_quantity_range,omitempty"`
+	PrerequisiteShippingPriceRange         *PrerequisiteShippingPriceRange         `json:"prerequisite_shipping_price_range,omitempty"`
+	PrerequisiteToEntitlementQuantityRatio *PrerequisiteToEntitlementQuantityRatio `json:"prerequisite_to_entitlement_quantity_ratio,omitempty"`
```

Comment:
I should investigate a non-breaking version of this PR, I think it should be possible just by adding a setter to the struct pointer for each field instead. I'll leave this open and we can choose which is best.